include/grpc/impl/codegen/port_platform.h

```diff
@@ -52,7 +52,7 @@
  * Defines GRPC_ERROR_IS_ABSEIL_STATUS to use absl::Status for grpc_error_handle
  */
 #ifndef GRPC_ERROR_IS_ABSEIL_STATUS
```

Comment:
Shouldn't we just remove this?

Reply:
That will be done sometime after this change is proven safe.

Reply:
Should we leave the configuration in BUILD then, to be able to quickly toggle it?

Reply:
Yeah, that's a good idea. I just reintroduced it again.